src/Common/src/System/Net/Logging/NetEventSource.Common.cs

```diff
@@ -111,7 +112,7 @@ public static void Enter(object thisOrContextObject, FormattableString formattab
         /// <param name="arg0">The object to log.</param>
         /// <param name="memberName">The calling member.</param>
         [NonEvent]
-        public static void Enter(object thisOrContextObject, object arg0, [CallerMemberName] string memberName = null)
+        public static void Enter(object thisOrContextObject, object arg0, [CallerMemberName] string? memberName = null)
```

Comment:
This file is shared amongst a bunch of libraries, and I think it's likely we'll need to come back to this and add more `?`s based on additional usage in those libraries, but I'm fine with this for now, especially since the type is internal.